lib/preprocess/raw.js

```diff
@@ -0,0 +1,55 @@
+const { nodeListToArray } = require('../utils')
+
+const regexp = /{{docxRaw [^{}]{0,500}}}/
+
+const supportedParentElements = ['w:r', 'w:p', 'w:tc']
+
+// the problem is that the {{docxRaw}} literal is in a w:t element, which is supposed to only contain literal text,
+// but we want the docxRaw helper to provide a raw XML fragment.
+// Word 365 is not bothered by that, but other docx editors can be.
+// E.g. Word Online displays a broken table and Libreoffice drops the run altogether.
+
+// we find the {{docxRaw}} literal in the w:t element and move it up the tree so it is in its desired location.
+
+module.exports = (files) => {
+  const documentFile = files.find(f => f.path === 'word/document.xml').doc
+  const generalTextElements = nodeListToArray(documentFile.getElementsByTagName('w:t'))
+
+  for (const textEl of generalTextElements) {
+    // there may be more than one docxRaw helper call in a single w:t
+    while (textEl.textContent.includes('{{docxRaw')) {
+      // TODO: do we want to support swapped parameters, i.e. xml after replaceParentElement?
+      const args = textEl.textContent.match(/{{docxRaw\s+xml=(?<xml>[^{}\s]+)\s+replaceParentElement="(?<replaceParentElement>[^{}\s]+)"/)

```

Comment:
The regex must be updated to accept single quotes literals for the second parameter.

Reply:
Done.